dbt_copilot_helper/commands/dns.py

```diff
@@ -545,10 +579,10 @@ def check_domain(domain_profile, project_profile, base_domain, env):
 @click.option(
     "--project-profile", help="AWS account profile name for application account", required=True
 )
-def assign_domain(app, domain_profile, project_profile, svc, env):
+def assign(app, domain_profile, project_profile, svc, env):
```

Comment:
`assign` feels a bit vague to me, maybe `add_load_balancer` or something?

Reply:
You can ignore my comment, I was seeing the assign out of the `copilot-helper domain assign-domain` context, where it makes more sense.